packages/aws-cdk-lib/aws-appconfig/lib/configuration.ts

```diff
@@ -62,7 +61,9 @@ export interface ConfigurationOptions {
    * The list of environments to deploy the configuration to.
    *
    * If this parameter is not specified, then there will be no
-   * deployment.
+   * deployment created alongside this configuration.
+   *
+   * A deployment can be added later via IEnvironment.
```

Comment:
```suggestion
   * Deployments can be added later using the `IEnvironment.addDeployment` or `IEnvironment.addDeployments` methods.
```
A bit more specific.